code/modules/mob/living/carbon/species.dm

```diff
@@ -418,6 +435,10 @@ var/global/list/whitelisted_species = list("Human")
 /datum/species/skellington/can_artifact_revive()
 	return 0
 
+/datum/species/skellington/gib(mob/living/carbon/human/H as mob)
+	H.dust()
```

Comment:
Dusting completely deletes the mob, doesn't it? Gibbing, on the other hand, leaves a head behind.
You may want to create a variant of `dust()` or give it arguments or something to allow it to pop the head off first, if you don't want gibbed skeletons to be inherently worse off than gibbed humans.

Reply:
I was supposed to do that BEFORE making this PR. I had forgotten.